src/components/ItaliaTheme/View/index.js

```diff
@@ -19,6 +25,7 @@ export TextOrBlocks from 'design-comuni-plone-theme/components/ItaliaTheme/View/
 export Sharing from 'design-comuni-plone-theme/components/ItaliaTheme/View/Commons/Sharing';
 export Actions from 'design-comuni-plone-theme/components/ItaliaTheme/View/Commons/Actions';
 export SideMenu from 'design-comuni-plone-theme/components/ItaliaTheme/View/Commons/SideMenu';
+export SideMenuByTitles from 'design-comuni-plone-theme/components/ItaliaTheme/View/Commons/SideMenuByTitles';
```

Comment:
Io non lo esporterei questo, sennò webpack te lo mette nel bundle anche quando non viene usato e appesantiamo le cose, anche se di poco. Cioè sulle carte potrebbe essere furbo a sufficienza da capire che poi non viene usato e potrebbe non metterlo, ma in verità non è così furbo, almeno in volto, lo sto vedendo mentre faccio il lavoro di ottimizzazione del core Volto, dove stiamo togliendo tutti i file index che riesportano i componenti.